server/src/gameMachine/gameReducer.ts

```diff
@@ -56,7 +73,23 @@ export function createGameReducer(initialState?: BaseUnifiedGame) {
             return state;
         }
       },
-
+      time: (state = { time: "day", count: 0 }, action) => {
+        switch (action.type) {
+          case "StartDay":
+            return {
+              time: "day",
+              count: state.count + 1,
+            };
+          case "StartNight": {
+            return {
+              time: "night",
+              count: state.count,
```

Comment:
I believe this will start state at night 0 on first night which is totally fine if we want days zero-indexed but something to remember.